cygnus-common/spec/SPECS/cygnus-common.spec

```diff
@@ -171,6 +171,11 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Wed May 11 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.18.0
```

Comment:
CNR file should be flushed

Reply:
Done in https://github.com/telefonicaid/fiware-cygnus/pull/2166/commits/ffae28036d7adefec44d63a2651950f8df0a33f8